cognite/client/_api/simulators/simulator_integrations.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from cognite.client._api_client import APIClient
+from cognite.client._constants import DEFAULT_LIMIT_READ
+from cognite.client.data_classes.simulators.filters import SimulatorIntegrationFilter
+from cognite.client.data_classes.simulators.simulators import (
+    SimulatorIntegration,
+    SimulatorIntegrationList,
+)
+from cognite.client.utils._experimental import FeaturePreviewWarning
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+    from cognite.client.config import ClientConfig
+
+
+class SimulatorIntegrationsAPI(APIClient):
+    _RESOURCE_PATH = "/simulators/integrations"
+
+    def __init__(self, config: ClientConfig, api_version: str | None, cognite_client: CogniteClient) -> None:
+        super().__init__(config, api_version, cognite_client)
+        self._warning = FeaturePreviewWarning(
+            api_maturity="General Availability", sdk_maturity="alpha", feature_name="Simulators"
+        )
+
+    def list(
+        self,
+        limit: int = DEFAULT_LIMIT_READ,
+        filter: SimulatorIntegrationFilter | dict[str, Any] | None = None,
+    ) -> SimulatorIntegrationList:
+        """`Filter simulator integrations <https://developer.cognite.com/api#tag/Simulator-Integrations/operation/filter_simulator_integrations_simulators_integrations_list_post>`_
+
+        List simulator integrations
+
+        Args:
+            limit (int): The maximum number of simulator integrations to return.
```

Comment:
@polomani since the maximum number of integrations a project can have is 20, should we lower the default max limit to 20 as well?

Reply:
@evertoncolling this part `Defaults to 1000.` is not correct here, since the method applies `limit: int = DEFAULT_LIMIT_READ`, which is 25.
so it's not the same as API could potentially give and seems like SDK default it quite nice (so the output readable for the user by default)